File: lib/src/main/java/com/sha/formvalidator/validator/pattern/EmailValidator.java
package com.sha.formvalidator.validator.pattern;

import android.util.Patterns;

/**
 * This validates an email using regexps.
 * Note that if an email passes the validation with this validator it doesn't mean it's a valid email - it means it's a valid email <storng>format</strong>
 *
 */
public class EmailValidator extends PatternValidator {

    public EmailValidator() {
        super(null, Patterns.EMAIL_ADDRESS);
    }

    public EmailValidator(String errorMessage) {
        super(errorMessage, Patterns.EMAIL_ADDRESS);
    }
}

File: lib/core/src/main/java/com/sha/formvalidator/core/validator/pattern/EmailValidator.kt
package com.sha.formvalidator.core.validator.pattern

import android.util.Patterns

/**
 * This validates an email using regexps.
 * Note that if an email passes the validation with this validator it doesn't mean it's a valid email - it means it's a valid email <storng>format
 *
</storng> */
class EmailValidator(errorMessage: String = "") : PatternValidator(errorMessage, Patterns.EMAIL_ADDRESS) {
}
